src/main/java/com/climatetree/places/dto/PlaceDTO.java

```diff
@@ -121,10 +125,35 @@ public boolean equals(Object obj) {
 		if (getClass() != obj.getClass())
 			return false;
 		PlaceDTO other = (PlaceDTO) obj;
+		if (Double.doubleToLongBits(carbon) != Double.doubleToLongBits(other.carbon))
```

Comment:
is there a reason for not doing a simple `==` check?

Reply:
we can do a simple equals check. This is just what eclipse generated for the equals function